SimPEG/Directives.py

```diff
@@ -313,6 +314,9 @@ class Update_IRLS(InversionDirective):
     coolingFactor = 2.
     coolingRate = 1
 
+    # store the l2model
+    l2model = None
```

Comment:
@fourndo: I have moved the storing of the `l2model` onto the IRLS directive instead of the regularization (eg to fetch the `l2model`, you now need to use 
```
IRLS = Directives.Update_IRLS(norms=([0, 1, 1, 1]),  eps=None, f_min_change=1e-2, minGNiter=3)
update_Jacobi = Directives.Update_lin_PreCond()
inv = Inversion.BaseInversion(invProb, directiveList=[IRLS, betaest, update_Jacobi])

inv.run(m)
l2model = IRLS.l2model
```
The `l2model` is not used by the regularization, and is quite specific to the use of this directive.